### DISCUSSION TITLE
How do I insert the `deno task` argument into the middle of that command?

### DISCUSSION THREAD

**Discussion by j-murata** [denoland/deno#25248] [Q&A]
The following `build` task is defined in `deno.json`.
I would like to change the output destination depending on the argument of `deno task build <arg>`, but I don't know how to do that.
```json
{
  "tasks": {
    "build": "deno compile -o ??? ./src/index.ts"
  }
}
```
Note that `deno compile` does not seem to work with the `-o` flag at the end, as shown below.
```json
{
  "tasks": {
    "build": "deno compile ./src/index.ts -o"
  }
}
```
If you have any good ideas, I would be happy to hear them.

**Comment by BlackAsLight** [comment]
Is `<arg>` a set of fixed, pre-determined values?

**Comment by j-murata** [comment]
Hi @BlackAsLight, thank you for your comment.

> Is `<arg>` a set of fixed, pre-determined values?

No, for now I assume that the output path can be determined by the person executing the `deno task build`.
For example, `./foo.exe`, `/tmp/bar.exe`, etc.

I also tried the following, but somehow `Deno.args[0]` seems to be `undefined`.
```json
{
  "tasks": {
    "build": "deno compile -o $(deno eval \"console.log(Deno.args[0])\") ./src/index.ts"
  }
}
```
